core/server/worker/src/main/java/alluxio/worker/grpc/BlockReadHandler.java

```diff
@@ -109,7 +107,7 @@ protected DataBuffer getDataBuffer(BlockReadRequestContext context, long offset,
       openBlock(context);
       BlockReader blockReader = context.getBlockReader();
       Preconditions.checkState(blockReader != null);
-      ByteBuf buf = PooledByteBufAllocator.DEFAULT.buffer(len, len);
+      ByteBuf buf = BufferAllocator.DEFAULT.buffer(len);
```

Comment:
What are the advantages of using your `BufferAllocator` over the `PooledByteBufAllocator` from netty?